app/lib/finder_api.rb

```diff
@@ -72,7 +65,7 @@ def relevance_scores_exist?(results)
     results.all? { |result| result['es_score'].present? }
   end
 
-  def fetch_search_response(content_item)
+  def fetch_search_response
```

Comment:
maybe keep the dependency on content_item - I'd be tempted to add filter_params as well